example/lib/screen/xayn_widgets/page/settings_page.dart

```diff
@@ -58,19 +61,19 @@ class _PrivacySectionState extends State<_PrivacySection> {
 
   @override
   Widget build(BuildContext context) => SettingsSection(
-        title: 'Privacy settings',
+        title: 'General information',
         items: [
-          _getBlockTrackers(),
-          _getBlockAds(),
-          _getCookies(),
+          _getAboutXayn(),
           _getEmptyCache(),
         ],
       );
 
+  // ignore: unused_element
   SettingsCardData _getBlockTrackers() {
     final tile = SettingsTileData(
```

Comment:
❓  whats about those ignored elements?

Reply:
I decided to leave those elements in the codebase. 
From the talk with Adriel, I got that right now they are busy and do not have time for implementing all components, but soon they will renew them as well. So I decided to leave them and later just uncomment and modify them.